File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-2/java/src/test/java/probe/ProjectTests.java
package probe;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class ProjectTests {
    @Test
    void scaleWorks() { assertEquals(6, ModA.scale(2)); }

    @Test
    void scaleAndOffsetWorks() { assertEquals(7, ModB.scaleAndOffset(2)); }
}

File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/TEST-2/cpp/project_tests.cpp
#include <iostream>
#include "mod_a.hpp"
#include "mod_b.hpp"
int main() {
  if (scale(2) != 6) { std::cout << "scale failed\n"; return 1; }
  if (scale_and_offset(2) != 7) { std::cout << "scale_and_offset failed\n"; return 1; }
  std::cout << "project tests passed\n";
  return 0;
}
